tests/python/frontend/tflite/test_forward.py

```diff
@@ -4572,6 +4650,47 @@ def test_forward_tflite_float16():
     tvm.testing.assert_allclose(tvm_sorted_labels, tflite_sorted_labels)
 
 
+def test_forward_tflite_mobilenet_int16():
+    """Test int16 quantized model"""
+    # MobilenetV2
+    tflite_model_file = tf_testing.get_workload_official(
```

Comment:
Nit: I suppose the model in this file is in TensorFlow, not TFLite?

Reply:
Correct, I actually simplified a bit the term overloading by making it now `model_file` and `test_forward_mobilenet_int16` as they are already part of the TFLite frontend test.